### PR TITLE
Add Django 1.9 and 1.10 compatability

### LINKED ISSUE #1: Attribute Error
Hi contributors

I am using Django with HANA and I am getting following error
AttributeError: 'SQLInsertCompiler' object has no attribute 'placeholder'
Django = 1.9 and HANA = SPS11
Is this version issue or any bug?

regards
Rahul

### PR DESCRIPTION
**Pull request by cmfcmf**
This should fix #1.
// @hefowe @hendraet @nilsthamm @philipp-bode

### CODE REVIEW

**Comment by cmfcmf** [comment]
Although #1 is fixed, it appears like there are additional issues with Django 1.10. The migrations currently seem unable to create tables. I will investigate further.

File: django_hana/compiler.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 from django.db.utils import DatabaseError
 from django.db import models
 from django.db.models.sql import compiler
+import django
```
Comment (mathebox) [inline django_hana/compiler.py]: Please move this to line 3.

File: django_hana/compiler.py [hunk 2]
```diff
@@ -31,6 +33,12 @@ def as_sql(self):
         return result, update_params
 
 class SQLInsertCompiler(compiler.SQLInsertCompiler, SQLCompiler):
+    def __createPlaceholder(self, field, val):
```
Comment (mathebox) [inline django_hana/compiler.py]: Can you create a `compat.py` similar to the approach of django rest framework? https://github.com/tomchristie/django-rest-framework/blob/master/CONTRIBUTING.md#managing-compatibility-issues

File: django_hana/compiler.py [hunk 3]
```diff
@@ -31,6 +33,12 @@ def as_sql(self):
...
+        if StrictVersion(django.get_version()) >= StrictVersion('1.9.0'):
```
Comment (mathebox) [inline django_hana/compiler.py]: I would prefer `django.VERSION >= (1, 9)`. Have a look at `compat.py` of DRF ;)

**Review by mathebox — CHANGES_REQUESTED** [review]
Thanks a lot for your PR. I would be great if you could fix the remaining issues with django 1.10

**Comment by cmfcmf** [comment]
I've now fixed a few more issues. Migrations run with Django 1.8, 1.9 and 1.10. I also incorporated a `compat.py` file. Please take a look @mathebox!

**Comment by cmfcmf** [comment]
Hey @mathebox, anything left I need to do before this can be merged?

**Review by mathebox — APPROVED** [review]

**Comment by mathebox** [comment]
Sorry, busy week..

LGTM

**Comment by cmfcmf** [comment]
Thanks Max!

**Pull request merged**